### PR TITLE
High-Impact Subdomain Takeover update

### PR DESCRIPTION
**Pull request by RRudder**

### CODE REVIEW

File: submissions/description/server_security_misconfiguration/misconfigured_dns/high_impact_subdomain_takeover/template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+A high-impact subdomain takeover vulnerability was identified which could impact the reputation and brand of the business. An attacker can register a subdomain on behalf of the target domain and use it for spamming and phishing attacks.
+
+## Business Impact
+
+High-Impact subdomain takeover could lead to data theft and indirect financial loss through the attacker’s ability to interact with legitimate users. These malicious actions could also result in reputational damage for the business through the impact to customers’ trust.
+
+## Steps to Reproduce
+
+1. Register DNS record using a registration service and point to the following endpoint: {{URL}}
```
Comment (evildaemond) [inline submissions/description/server_security_misconfiguration/misconfigured_dns/high_impact_subdomain_takeover/template.md:15]: Step can be removed

File: submissions/description/server_security_misconfiguration/misconfigured_dns/high_impact_subdomain_takeover/template.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# High-Impact Subdomain Takeover
+
+## Overview of the Vulnerability
+
+A subdomain takeover is when a misconfigured Domain Name System (DNS) record is re-registered to an endpoint owned by an attacker. An attacker is then able to redirect users to the endpoint and capture data such as cookies and credentials, perform Cross-Site Scripting (XSS) attacks, and potentially take over accounts in the legitimate application.
+
+A high-impact subdomain takeover vulnerability was identified which could impact the reputation and brand of the business. An attacker can register a subdomain on behalf of the target domain and use it for spamming and phishing attacks.
```
Comment (evildaemond) [inline submissions/description/server_security_misconfiguration/misconfigured_dns/high_impact_subdomain_takeover/template.md:7]: Highlighting the technical impact will help clarify what the difference here is, at the end of the day we are trying to prove the researcher and customer a direct explanation of what a High Impact subdomain takeover.

**Review by evildaemond — CHANGES_REQUESTED** [review]
2 changes then LGTM

File: submissions/description/server_security_misconfiguration/misconfigured_dns/high_impact_subdomain_takeover/template.md (hunk 2 above)
Comment (RRudder) [reply submissions/description/server_security_misconfiguration/misconfigured_dns/high_impact_subdomain_takeover/template.md]: @evildaemond Please let me know if the additional context is sufficient

File: submissions/description/server_security_misconfiguration/misconfigured_dns/high_impact_subdomain_takeover/template.md (hunk 1 above)
Comment (RRudder) [reply submissions/description/server_security_misconfiguration/misconfigured_dns/high_impact_subdomain_takeover/template.md]: removed, ty

**Review by evildaemond — APPROVED** [review]
LGTM!

**Pull request merged**